app/models/google_apps/auth/authorization.rb

```diff
@@ -0,0 +1,108 @@
+module GoogleApps
+  module Auth
+    # Provides main interface for Google App Authorization process
+    class Authorization
+      include ClassLogger
+
+      # Returns authorization object used with Google API requests
+      def self.refresh_credential(opts = {})
+        # Google::Auth::WebUserAuthorizer#get_credentials requires a 'request' argument
+        # so that it can detect and save any updated token data if included with the request.
+        # CalCentral shouldn't need to provide the HTTP request object,
+        # just to get the 'authorization' object for a Google HTTP API request
+        # This assembles such an object similar to the Google::Auth library.
+        opts.reverse_merge!({
+          access_token: 'fake_access_token',
+          refresh_token: 'fake_refresh_token',
```

Comment:
Why include "fake" defaults?

Reply:
For the test environment. Want them there so that the contract is not broken, but don't expect it to actually try to make the call.